### PR TITLE
Add warnning message in case some seed are not reachable

### LINKED ISSUE #5913: Show warning if Seed Cluster is not reachable, but don't block cluster loading
**What is the problem, where is it and why is it an issue?**
Currently if a seed cluster is not available from the master, we have
- `2.21`: cluster searching is hanging and dashboard is blocked, see #5108 
- `2.22`:  cluster don't show up

**Suggest how to fix it, if possible**
PROD' 2023-04-24 15-01-21

In the above example we manage in one Project multiple cluster on different regions, so a UI user want to get a warning if some of them are not reachable (via the seed)

### PR DESCRIPTION
**Pull request by ahmadhamzh**
**What this PR does / why we need it**:
Add warnning message in the cluster list page in case some seeds are not reachable .

**Cluster list**:
image

**Which issue(s) this PR fixes**:

Fixes #5913 

**What type of PR is this?**
/kind design

```release-note
Add warning message in the cluster list page in case some seeds are not reachable.
```

```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/app/cluster/list/cluster/template.html [hunk 1]
```diff
@@ -14,11 +14,20 @@
...
     <mat-card-title>Clusters</mat-card-title>
   </mat-card-header>
-  <div fxLayout
+  <div *ngIf="!!projectClusterListErrorMessage"
+  fxLayoutAlign=" center"
+  class="error-message">
+<i class="km-icon-warning"></i>
+<div>{{projectClusterListErrorMessage}}
+  <a *ngIf="isAdmin"
+  href="settings/seeds">check Seed Configration</a></div>
```
Comment (Waseem826) [inline modules/web/src/app/cluster/list/cluster/template.html:28]:
```suggestion
  href="settings/seeds">Check Seed Configuration</a></div>
```

File: modules/web/src/app/cluster/list/cluster/template.html [hunk 2]
```diff
@@ -14,11 +14,20 @@
...
+</div>
+  <div
```
Comment (Waseem826) [inline modules/web/src/app/cluster/list/cluster/template.html:30]: Please run `fix:html` script to format these changes.

File: modules/web/src/app/cluster/list/cluster/template.html (hunk 1 above)
Comment (Waseem826) [reply modules/web/src/app/cluster/list/cluster/template.html:28]: Is this link highlighted (blue color and pointer cursor on hover) like other links in our application?

**Review by Waseem826 — CHANGES_REQUESTED** [review]
Some feedback. PTAL

File: modules/web/src/app/cluster/list/cluster/template.html (hunk 1 above)
Comment (ahmadhamzh) [reply modules/web/src/app/cluster/list/cluster/template.html:28]: yes i just update the pic in the description above

File: modules/web/src/app/cluster/list/cluster/component.ts [hunk 3]
```diff
@@ -142,14 +147,19 @@ export class ClusterListComponent implements OnInit, OnChanges, OnDestroy {
           this._etcdRestores = restores;
         })
       )
-      .pipe(switchMap(_ => this._clusterService.clusters(this._selectedProject.id)))
-      .pipe(tap(clusters => (this.clusters = clusters)))
+      .pipe(switchMap(_ => this._clusterService.projectClusterList(this._selectedProject.id)))
+      .pipe(
+        tap(projectCluster => {
+          this.projectClusterListErrorMessage = 'Failed to fetch data for following seeds: asia, europe.';
```
Comment (Waseem826) [inline modules/web/src/app/cluster/list/cluster/component.ts:153]: I guess you forgot to revert this ;)

**Review by Waseem826 — APPROVED** [review]

**Comment by ahmedwaleedmalik** [comment]
/lgtm cancel

Please add a release note

**Review by Waseem826 — APPROVED** [review]

**Pull request merged**